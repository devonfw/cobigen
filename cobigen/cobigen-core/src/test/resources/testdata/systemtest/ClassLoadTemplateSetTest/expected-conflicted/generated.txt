Testing JarredClass:
String is 2