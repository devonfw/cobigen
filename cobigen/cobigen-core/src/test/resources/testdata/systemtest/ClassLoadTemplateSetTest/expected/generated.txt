Testing JarredClass:
String is Something
5
generator